base/debian/cuttlefish-base.cuttlefish-host-resources.init

```diff
@@ -228,15 +243,33 @@ start() {
     echo 1 > /proc/sys/net/ipv4/ip_forward
     echo 1 > /proc/sys/net/ipv6/conf/all/forwarding
 
+    # Ethernet
+    create_bridged_interfaces \
+      192.168.98 "${ethernet_bridge_interface}" cvd-etap \
+      "${ethernet_ipv6_prefix}" "${ethernet_ipv6_prefix_length}"
+
+    # Mobile Network
+    for i in $(seq ${num_cvd_accounts}); do
+        tap="$(printf cvd-mtap-%02d $i)"
+        if [ $i -lt 65 ]; then
+            create_interface $tap 192.168.97 $i
+        elif [ $i -lt 129 ]; then
+            create_interface $tap 192.168.93 $(($i - 64))
+        fi
+    done
+
+    # Wireless Network
     create_bridged_interfaces \
       192.168.96 "${wifi_bridge_interface}" cvd-wtap \
       "${wifi_ipv6_prefix}" "${wifi_ipv6_prefix_length}"
     for i in $(seq ${num_cvd_accounts}); do
```

Comment:
please add comment about the difference between wtap and wifiap

Reply:
Done.